plugin/src/sidebar.ts

```diff
@@ -353,6 +353,16 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
     }
   }
 
+  // ************************************************
+  // Style code
+  else if (action === "styleCodeUpdate") {
```

Comment:
I don't think you need this route. The "updateDatasetInScene" action is appropriate, right?

Reply:
Ah yes i can use that. 
And i found it get the layerID twice in "updateDatasetInScene" and removed one.

Reply:
You seem to have remoed the double layerId thing but not removed this route. Please remove this.